setup/air-gapped/index.md

```diff
@@ -50,13 +50,13 @@ Throughout this article, we will suggest changes to the Helm chart that you'll
 obtain from the `.tgz` that's returned when you run `helm pull`. We recommend
 version controlling your files.
 
-## Step 1: Pull all Coder resources into your air-gapped workspace
+## Step 1: Pull all Coder resources into your air-gapped environment
 
 Coder is deployed through [helm](https://helm.sh/docs/intro/install/), and the
 platform images are hosted in Coder's Docker Hub repo.
 
 1. Pull down the Coder helm charts by running the following in a non-air-gapped
```

Comment:
perhaps we could phrase this as "Download the Coder Helm package by running the following command outside of the air gap:" or similar